caffe2/python/operator_test/rnn_cell_test.py

```diff
@@ -36,17 +36,20 @@
 import unittest
 
 
-def lstm_unit(hidden_t_prev, cell_t_prev, gates,
-              seq_lengths, timestep, forget_bias=0.0,
-              drop_states=False, no_sequence_lengths=False):
+def lstm_unit(*args, **kwargs):
```

Comment:
why not make seq_lengths be None in a case it is not provided? Then you don't need this kwargs parsing. I.e. you would need just to rearrange inputs order here a bit, put seq_lengths in the optional section

Reply:
not possible - the interface here has to exactly match the interface of the C++ op, I can't choose where to put the args

Reply:
yeah, makes sense